ex7/index.js

```diff
@@ -0,0 +1,21 @@
+const prompt = require("prompt-sync")({ sigint: true});
+
+console.log("Insert the number of apples purchased:");
+const apples = Number(prompt(">"));
+
+const DEFAULT_PRICE = 0.30;
+const DISCOUNT_PRICE = 0.25;
+const DISCOUNT_QUANTITY = 12;
+
+function shop() {
+    if (apples < DISCOUNT_QUANTITY) {
+        const finalPrice = (apples * DEFAULT_PRICE).toFixed(2);
+        return (`Your purchase will cost $${finalPrice} dollars.`);
+    } 
+    else {
```

Comment:
Since you have a `return` inside the `if` statement, the `else` becomes unnecessary based on the early return pattern (see https://gomakethings.com/the-early-return-pattern-in-javascript/ and https://dev.to/dicas_devon/dicas-para-o-uso-do-padrao-early-return-no-javascript-3bb7)